src/assets/alignmentD3Tests.js

```diff
@@ -177,75 +230,70 @@ var isFeatureAligned = function(featureCoord, axis, tickPxCur, tickPxNext) {
   // (inclusive) the two ticks' axis coordinates, plus-or-minus 1px.
   // Codepen and regular browser had 0.5px differences for these coordinates,
   // hence the 1px buffer.
-  return featureCoord >= (minTickCoord - 1) && featureCoord <= (maxTickCoord + 1)
+  return featureCoord >= minTickCoord - 1 && featureCoord <= maxTickCoord + 1
 };
 
 /**
  * For given tick i and i+1, get features whose associated values are
  * between (inclusive) tick i and i+1 values and runs isFeatureAligned on each.
+ * @param {Object} axis - object created from getAxisInfo
+ * @param {Array} collection - HTMLCollection of features
+ * @param {Function} getFeatureValueFunc - via getMisalignmentCountCaller
  * @param {Number} tickPxCur - Coordinate (x or y) for tick[i]
  * @param {Number} tickPxNext - Coordinate (x or y) for tick[i+1]
  * @param {Number} tickValCur - parsed from getTickValue(getTickText())
  * @param {Number} tickValNext - parsed from getTickValue(getTickText())
- * @param {Object} axis - object created from getAxisInfo
- * @param {Array} collection - HTMLCollection of features
- * @param {Function} getFeatureValue - (x or y) via getMisalignmentCountCaller
- * @param {Function} getFeatureCoord - (x or y) via getMisalignmentCountCaller
  * @return {Number} Count of misaligned features via isFeatureAligned
 */
 var getMisalignmentCount = function(
+  axis,
+  collection,
+  getFeatureValueFunc,
   tickPxCur,
   tickPxNext,
   tickValCur,
-  tickValNext,
-  axis,
-  collection,
-  getFeatureValue,
-  getFeatureCoord
+  tickValNext
 ) {
   var count = 0;
 
   for (var j = 0; j < collection.length - 1; j++) {
-      // get values for given feature (j)
-      var featureVal = getFeatureValue(collection.item(j)),
-          featureCoord = getFeatureCoord(collection.item(j));
-      if (featureVal >= tickValCur && featureVal <= tickValNext) {
-          if (!isFeatureAligned(featureCoord, axis, tickPxCur, tickPxNext)) {
-            count++;
-          }
+    // get values for given feature (j)
+    var featureVal = getFeatureValueFunc(collection.item(j), axis.dataAttr),
+      featureCoord = getFeatureCoord(collection.item(j), axis.coordAttr);
+    if (featureVal >= tickValCur && featureVal < tickValNext) {
+      if (!isFeatureAligned(featureCoord, axis, tickPxCur, tickPxNext)) {
+        console.log(collection.item(j), axis.coordAttr, featureCoord, tickPxCur, tickPxNext)
```

Comment:
`console.log` should be removed, unless it serves a production purpose. This looks like debugging?